frontend/docsettings.lua

```diff
@@ -412,6 +418,7 @@ function DocSettings:purge(sidecar_to_keep, data_to_purge)
             util.removePath(self.hash_sidecar_dir) -- remove empty parent folders
         end
     end
+    DocSettings.setIsHashLocationEnabled() -- reset this in case last hash book is purged
```

Comment:
Given the new name of this function, it feels like it would read better if you write/provide explicitely `nil`:
`DocSettings.setIsHashLocationEnabled(nil)`
(Otherwise, it reads as you would set (to true) is_hash_location_enabled).